### PR TITLE
Redesign my account

### PR DESCRIPTION
**Pull request by atachh**
Mobile and desktop styling

### CODE REVIEW

File: src/app/route/MyAccountDetails/MyAccountDetails.style.scss [hunk 1]
```diff
@@ -1,128 +1,151 @@
...
+                    @include mobile {
+                        font-size: .9rem;
+                    }
+                }
+            }
         }
     }
 
-    form {
+    .Form {
```
Comment (alfredsgenkins) [inline src/app/route/MyAccountDetails/MyAccountDetails.style.scss]: Can we make form accept mix?

File: src/app/route/MyAccountDetails/MyAccountDetails.style.scss [hunk 2]
```diff
@@ -1,128 +1,151 @@
...
         display: grid;
         grid-template-columns: 45% 45%;
-        grid-column-gap: 10%;
+        grid-column-gap: 5%;
```
Comment (alfredsgenkins) [inline src/app/route/MyAccountDetails/MyAccountDetails.style.scss]: I do not think that 5% gap is good, fixed gaps are better.

**Comment by alfredsgenkins** [comment]
Please share screenshots of:
- My account overlay on desktop in states: sign in, created account, forgot password, logged in.
- Please make sure logging in with wrong details do not freeze a loader.
- Please style notification so it appears along bottom of the header on both desktop and mobile.

File: src/app/component/MyAccountOverlay/MyAccountOverlay.component.js [hunk 3]
```diff
@@ -437,7 +456,14 @@ MyAccountOverlay.propTypes = {
     showNotification: PropTypes.func.isRequired,
     createAccount: PropTypes.func.isRequired,
     logout: PropTypes.func.isRequired,
+    isOverlayVisible: PropTypes.bool.isRequired,
```
[the remark this answers is not in the document]
Comment (atachh) [reply src/app/component/MyAccountOverlay/MyAccountOverlay.component.js]: this prop is used inside getDerivedStateFromProps

**Pull request merged**